src/components/horizontal_rule/horizontal_rule.tsx

```diff
@@ -22,16 +35,18 @@ const marginToClassNameMap = {
 
 export const MARGINS = Object.keys(marginToClassNameMap);
 
-export const EuiHorizontalRule = ({
+export const EuiHorizontalRule: SFC<
+CommonProps & HTMLAttributes<HTMLHRElement> & EuiHorizontalRuleProps
+> = ({
   className,
-  size,
-  margin,
+  size = 'full',
```

Comment:
I'd choose either defaulting these values here _or_ using the ternary later, not both.

Reply:
oh, sorry, meant to remove this. Thanks

Reply:
OK. Addressed.